### PR TITLE
[IMP] sale: move_field from sale_subscription to sale

### PR DESCRIPTION
**Pull request by fdamhaut**
Move some KPI related fields from sale_subscription to sale

task-id : 3244560

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/models/sale_order.py [hunk 1]
```diff
@@ -262,6 +262,8 @@ def _rec_names_search(self):
         comodel_name='crm.tag',
         relation='sale_order_tag_rel', column1='order_id', column2='tag_id',
         string="Tags")
+    country_id = fields.Many2one('res.country', related='partner_id.country_id', store=True, compute_sudo=True)
+    industry_id = fields.Many2one('res.partner.industry', related='partner_id.industry_id', store=True)
```
Comment (Feyensv) [inline addons/sale/models/sale_order.py:266]:
```suggestion
    country_id = fields.Many2one(related='partner_id.country_id', store=True, compute_sudo=True)
    industry_id = fields.Many2one(related='partner_id.industry_id', store=True)
```

Also, those fields should be in the `partner-based computes` section.

**Review by Feyensv — COMMENTED** [review]
Well, i understand that there was a todo about those fields in `sale_subscription`, but why ?
If those fields are not shown/used in sale, why should we add and store them here ? :thinking:

**Comment by fdamhaut** [comment]
> Well, i understand that there was a todo about those fields in `sale_subscription`, but why ? If those fields are not shown/used in sale, why should we add and store them here ? thinking

FP-request for reporting

+ Will change the position :D

**Comment by fdamhaut** [comment]
Update : the fields are already present in reporting, dala agree they are not needed in the SO view  -> they will not be added to sale anymore

**Pull request closed without merging**